Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.cpp

```diff
@@ -327,8 +419,10 @@ namespace O3DE::ProjectManager
         {
             // Add all available gems to the model.
             const QVector<GemInfo>& allGemInfos = allGemInfosResult.GetValue();
-            for (const GemInfo& gemInfo : allGemInfos)
+            for (GemInfo gemInfo : allGemInfos)
             {
+                // Mark as downloaded because this gem was registered with an existing directory
+                gemInfo.m_downloadStatus = GemInfo::DownloadStatus::Downloaded;
```

Comment:
I wonder, should we be modifying the downloadStatus here or in GetAllGemInfos()

Reply:
Oh yea that makes more sense. I forgot there was a step in between I could insert it.